code/modules/mob/living/carbon/alien/humanoid/emote.dm

```diff
@@ -8,8 +8,23 @@
 //	if(findtext(act,"s",-1) && !findtext(act,"_",-2))//Removes ending s's unless they are prefixed with a '_'
 //		act = copytext(act,1,length(act))	//seriously who the fuck wrote this
 	var/muzzled = is_muzzled()
+
+	var/on_CD = 0
```

Comment:
This won't work. The on_CD var is declared within the emote proc, it will be created and destroyed every time it's called, and therefore either never allow flipping to work or never stop flipping from working.